docs/how-to-test.md

```diff
@@ -115,31 +116,31 @@ Using Gravitino IT Docker container to run all integration tests. [deploy test]
 -----------------------------------------------------------------
 ```
 
-Only when all the required environments are ready, all the integration tests run, otherwise,
-only parts of the integration tests without `gravitino-docker-it` tag run.
+All the integration tests only run when all the required environments are ready. Otherwise,
+only the parts of the integration tests without the `gravitino-docker-it` tag run.
 
 ## How to debug Gravitino server and integration tests in embedded mode
 
-By default, the integration tests run in the embedded mode, `MiniGravitino` starts in the same process. Debugging `MiniGravitino` is simple and easy. You can modify any code in the Gravitino project and set breakpoints anywhere.
+By default, the integration tests run in the embedded mode. `MiniGravitino` starts in the same process. Debugging `MiniGravitino` is simple and easy. You can modify any code in the Gravitino project and set breakpoints anywhere.
```

Comment:
the integration tests run in the embedded mode, and `MiniGravitino` starts in the same process.